Introduction-to-Grails-Development-with-Intellij/index.md

```diff
@@ -0,0 +1,293 @@
+### Getting Started with Grails Java Framework
+
+In recent years, many developers are using frameworks when coming up with websites. Frameworks have assisted in making sure there is  faster application deployment and also coming up with quality code. Java supports multiplatform including a desktop application, web application, mobile app development, and many more. In the web application, it can be embedded with Javascript, HTML, CSS, and many front end applications.
+
+### Introduction
+
+**Grails** is Java and Groovy framework developing agile web applications. Grails implements MVCS (Model, View, and Controller) design pattern.
+
+Grails is superb because it makes developers concentrate on application requirements and use less time in configuring the framework. Grails provides tools for development and it is build based on tools like  Quarts, Hibernate, Spring and uses Gradle for library management.
+
+There are many Grails framework version being built which has the support of different libraries and tools
+
+### Prerequisites
+
+- IntelliJ integrated development environment(IDE)
+- Knowledge in Java development
+- Java development kit(JDK) installed in your computer/laptop to help in application development
+
+### Installing Grails
+
+- Download [Grail Framework](https://grails.org/download.html) as per the tutorial we were at version 4.0.6 in which you will get a zipped file.
+- Extract the zipped file and place it where you keep development tools. This is how to do Windows [Unzip](https://www.windowscentral.com/how-zip-and-unzip-files-windows-10). Ubuntu [unzip](https://askubuntu.com/questions/499807/how-to-unzip-tgz-file-using-the-terminal)
+- Set environment variable `GRAILS_HOME` to point where Grails installation directory is located. [Set windows environment variable](https://www.computerhope.com/issues/ch000549.htm).[ Set Ubuntu environment variable](https://askubuntu.com/questions/58814/how-do-i-add-environment-variables)
+
+### Getting Started
+
+Grails provides a set of commands to support application development.IDE has also easen the use of commands and provided a way users can click menus for fast application development.
+
+To create an application in Grail framework `create-app` command can be passed in terminal or windows CMD with arguments like the name of the application.
```

Comment:
Consider rewriting "... Grail framework `create-app` …" to "... Grails framework `create-app`..."

Reply:
sorted